cmd/serv.go

```diff
@@ -290,17 +291,21 @@ func runServ(c *cli.Context) error {
 		return nil
 	}
 
-	// Special handle for Windows.
-	if setting.IsWindows {
-		verb = strings.Replace(verb, "-", " ", 1)
-	}
-
 	var gitcmd *exec.Cmd
-	verbs := strings.Split(verb, " ")
-	if len(verbs) == 2 {
-		gitcmd = exec.CommandContext(ctx, verbs[0], verbs[1], repoPath)
-	} else {
-		gitcmd = exec.CommandContext(ctx, verb, repoPath)
+	gitBinPath := filepath.Dir(git.GitExecutable) // e.g. /usr/bin
```

Comment:
`git.GitExecutable` maybe a relative path. e.g. just `git`.

Reply:
No, it can't.

![image](https://user-images.githubusercontent.com/2114189/221582526-6a5543d1-2b3a-4a02-8eb3-905f830049f1.png)